docs/develop/get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md

```diff
@@ -43,6 +43,84 @@ The manifest file for an Office Add-in provides important information about the
 
 1. In a text editor, add the following code to the manifest file.
 
+    # [Unified manifest for Microsoft 365 (preview)](#tab/jsonmanifest)
+
```

Comment:
Because the unified manifest is GAed for Outlook, we're not adding "(preview)". Instead, add a note explaining where it is and isn't preview, like this:

```suggestion
    # [Unified manifest for Microsoft 365 (preview)](#tab/jsonmanifest)

> [!NOTE]
> The unified manifest is generally available for production Outlook Add-ins. It is available only for preview in Excel, PowerPoint, and Word Add-ins.
```